__tests__/integration/create-token.test.js

```diff
@@ -0,0 +1,290 @@
+import { getRandomInt, TestUtils, WALLET_CONSTANTS } from './utils/test-utils-integration';
+import { WalletHelper } from './utils/wallet-helper';
+
+describe('create token', () => {
+  let wallet1;
+  let wallet2;
+
+  const tokenA = {
+    name: 'Token A',
+    symbol: 'TKA',
+    uid: null
+  };
+
+  beforeAll(async () => {
+    wallet1 = new WalletHelper('create-token-1');
+    wallet2 = new WalletHelper('create-token-2');
+
+    await WalletHelper.startMultipleWalletsForTest([wallet1, wallet2]);
+    await wallet1.injectFunds(10, 0);
+    await wallet1.injectFunds(10, 1);
+    await wallet2.injectFunds(10, 0);
+  });
+
+  afterAll(async () => {
+    await wallet1.stop();
+    await wallet2.stop();
+  });
+
+  // Testing failures first, that do not cause side-effects on the blockchain
+
+  it('should reject missing name parameter', async done => {
+    const response = await TestUtils.request
+      .post('/wallet/create-token')
+      .send({
+        symbol: tokenA.symbol,
+        amount: 1000
+      })
+      .set({ 'x-wallet-id': wallet1.walletId });
+
+    expect(response.status).toBe(400);
+    expect(response.body.success).toBe(false);
+    done();
+  });
+
+  it('should reject missing symbol parameter', async done => {
+    const response = await TestUtils.request
+      .post('/wallet/create-token')
+      .send({
+        name: tokenA.name,
+        amount: 1000
+      })
+      .set({ 'x-wallet-id': wallet1.walletId });
+
+    expect(response.status).toBe(400);
+    expect(response.body.success).toBe(false);
+    done();
+  });
+
+  it('should reject a name with more than 30 characters', async done => {
+    const response = await TestUtils.request
+      .post('/wallet/create-token')
+      .send({
+        name: 'Name input with more than 30 characters',
+        symbol: tokenA.symbol,
+        amount: 1000
+      })
+      .set({ 'x-wallet-id': wallet1.walletId });
+
+    expect(response.body.success).toBe(false);
+    expect(response.body.error).toContain('maximum size');
+    done();
+  });
+
+  // The result is an error with the message "maximum size", but consumes the funds. Must be fixed.
```

Comment:
On every debug I made, there were no other transactions executed on the fullnode. So, part of the test works as intended.
However, every transaction made after it triggers a "Insuficcient Funds" response.

This leads me to conclude that there is an error on the internal wallet-lib storage that happens and, since there are no websocket updates because the transaction did not happen, this mistake is not corrected.

As we plan to open an issue solving every skipped test on this suite, I understand that this bug is already mapped.

Reply:
I've just checked the code and you are correct. It's a local control and a bug. An issue should be created to fix this (as you said, we will go through all skipped tests (when are we going to do this?)

Reply:
Looking at the code, the same problem should happen with the name, are you sure it's not?

Reply:
You are right. I increased the "name" test amount to consume all the funds and the other tests started failing.

Probably the reason the error was not being triggered before is because at least one small transaction was being made successfully before the bugged funds were consumed, which updated the balance via websocket and "hid" the error.

Reply:
So we should skip it as well and in the issue explain that both tests refer to the same problem